dev/io.openliberty.cdi.4.0.internal_fat/fat/src/io/openliberty/cdi40/internal/fat/bce/enhance/beans.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://jakarta.ee/xml/ns/jakartaee"
+    xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd"
+    version="4.0" bean-discovery-mode="annotated">
```

Comment:
maybe make this beans.xml as empty one to cover the fact of empty beans.xml is the same as the one with `bean-discovery-mode="annotated"` specified?

Reply:
The change in empty beans.xml behaviour will be tested along with the new config option... So slightly more value in making sure that `beans_4_0.xsd` works OK?

Reply:
The new configuration does not exist yet so it is currently hard-coded to `ALL`.

Reply:
In CDI 4.0, by default, empty beans.xml should behave same as if annotated. It should not hard coded to `ALL` for cdi 4.0. For CDI 4.0, the default is `Annoated`. The new config can change the value to `All`.